calcom/cal.com
# Bug - teams being double created without slugs

## What does this PR do?



This PR aims to prevent users from accidentally double creating teams. On the team creation page, if a user double clicks on continue or comes back to the page then a new record will be created in the DB. If they go through a checkout session once, the team upgrade banner will continue to appear for all duplicated teams.

Fixes # (issue)



**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- On the team creation page, double click the continue button. Only one record should be made in the DB
- On the team creation page, click continue and then go back and create the same team. There should be two records now in the DB

## Checklist



- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
